specification/azsadmin/resource-manager/backup/readme.azsautogen.md

```diff
@@ -0,0 +1,50 @@
+# Backup Admin
```

Comment:
Just curious, what's the difference between this azsautogen one and the original readme.md?

Reply:
It was meant to carry the autorest gen info. Now all the info have been moved to readme.md. and removed azsautogen.md

Reply:
Got it. Then do we need to update PowerShell generation side also? https://github.com/Azure/azurestack-powershell/blob/master/src/Azs.Backup.Admin/readme.md
Currently this points to readme.azsautogen.md.

Actually I'm also fine if we have this azsautogen.md, as long as we align with other Azs RPs.